src/worker/dump_db.rs

```diff
@@ -246,6 +250,22 @@ impl Drop for DumpTarball {
     }
 }
 
+fn invalidate_caches(env: &Environment, target_name: &str) -> Result<(), PerformError> {
```

Comment:
```suggestion
fn invalidate_caches(env: &Environment, target_name: &str) {
```

since we don't use `?` anymore, this fn can't fail anymore :)